nav2_bt_navigator/behavior_trees/navigate_w_recovery_and_replanning_only_if_path_becomes_invalid.xml

```diff
@@ -10,8 +10,11 @@
       <PipelineSequence name="NavigateWithReplanning">
```

Comment:
Lets make this into another file, and that way we can rename that file. This file says `only` but if we add in the occasionally replanning, this name no longer describes what it is that we have for real. 

`navigate_w_recovery_and_constant_replanning_and_if_path_becomes_invalid.xml`